pkgs/servers/peertube/default.nix

```diff
@@ -127,7 +141,7 @@ in stdenv.mkDerivation rec {
     '';
     license = licenses.agpl3Plus;
     homepage = "https://joinpeertube.org/";
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
It probably works on aarch64-linux, too, right? So we can set this to platforms.unix.

Reply:
I tried to build this for aarch64, but I believe it would require node.bcrypt.js 5.1.0 because bcrypt_lib-v5.1.0-napi-v3-linux-arm64-glibc.tar.gz exists, but there is no bcrypt_lib-v5.0.1-napi-v3-linux-arm64-glibc.tar.gz: https://github.com/kelektiv/node.bcrypt.js/releases . However, even the current version of peertube (4.3.0) requires node.bcrypt.js 5.0.1.

Reply:
Now that peertube uses bcrypt 5.1.0, I added `aarch64-linux` although I am not sure how to test it ([per my comment](https://github.com/NixOS/nixpkgs/pull/194544#issuecomment-1616351339)).

Reply:
I kept the platforms listing the individual architectures and OSes as I noticed that `platforms.unix` seems to be even broader (including BSD, SunOS, etc.).